front_end/html/problem.html

```diff
@@ -231,6 +235,21 @@ <h2>Expected output:</h2>
 
             {% if len(problem_details["credit"]) > 0 %}
               <p><em>{{ problem_details["credit"] }}</em></p>
+            {% end %}  
+
+            <h2>Past Submissions:</h2>
```

Comment:
To be consistent with the other section headings, please use sentence casing (don't capitalize the "S" in Submissions).

Reply:
Done!